pages/authorDashboard/PKPAuthorDashboardHandler.inc.php

```diff
@@ -157,23 +159,34 @@ function setupTemplate($request) {
 			if ($fileStage && is_a($lastReviewRound, 'ReviewRound')) {
 				$editorDecisions = DAORegistry::getDAO('EditDecisionDAO')->getEditorDecisions($submission->getId(), $submission->getData('stageId'), $lastReviewRound->getId());
```

Comment:
My fault not yours, but `$lastReviewRound->getId()` should be `$lastReviewRound->getRound()`.

Reply:
check